news/_posts/2020-09-17-bc-break-part-1.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title: "A deeper look at Backward Compatibility Changes"
+subtitle: "About dependencies, API interfaces, SemVer and Backward Compatibility Changes"
+date:   2020-09-17
+authors: [ MathieuFerment ]
+image: /assets/images/theme/meta-logo-build.png
+icon: icon-code
+tags: [contribution, opensource, development]
+---
+
+In 2011, Marc Andreessen famously "Software is eating the world".
```

Comment:
maybe a verb is missing here

Reply:
Fixed in https://github.com/PrestaShop/prestashop.github.io/pull/777/commits/1a698888de149cecf0416f35ec952bbfdbec852a